sdk/storage/azure-storage-files-shares/test/share_client_test.cpp

```diff
@@ -213,6 +217,24 @@ namespace Azure { namespace Storage { namespace Test {
     EXPECT_EQ(ret2.Value.SignedIdentifiers, identifiers);
   }
 
+  TEST_F(FileShareClientTest, ShareAccessPolicyNullable)
+  {
+    std::vector<Files::Shares::Models::SignedIdentifier> identifiers;
+    for (unsigned i = 0; i < 3; ++i)
+    {
+      Files::Shares::Models::SignedIdentifier identifier;
+      identifier.Id = RandomString(64);
+      identifier.Policy.Permission = "r";
+      identifiers.emplace_back(identifier);
+    }
```

Comment:
More test cases, such as `StartsOn` is null, `Expires` is null, both of them are null.

Reply:
Added.